Solutions/FlareSystemsFirework/Package/createUiDefinition.json

```diff
@@ -60,34 +60,80 @@
             "name": "dataconnectors1-text",
             "type": "Microsoft.Common.TextBlock",
             "options": {
-              "text": "This Solution installs the data connector for FlareSystemsFirework. You can get FlareSystemsFirework custom log data in your Azure Sentinel workspace. Configure and enable this data connector in the Data Connector gallery after this Solution deploys. This data connector creates custom log table(s) Firework_CL in your Azure Sentinel / Azure Log Analytics workspace."
+              "text": "This Solution installs the data connector for FlareSystemsFirework. You can get FlareSystemsFirework custom log data in your Microsoft Sentinel workspace. Configure and enable this data connector in the Data Connector gallery after this Solution deploys. This data connector creates custom log table(s) Firework_CL in your Microsoft Sentinel / Azure Log Analytics workspace."
```

Comment:
Update the data connector text with the following Sample format.
This solution installs the data connector for ingesting Armorblox logs into Microsoft Sentinel, using the Armorblox API. After installing the solution, configure and enable this data connector by following guidance in Manage solution view.

Reply:
Please update the text for workbook as well